packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.styles.ts

```diff
@@ -330,8 +333,8 @@ export const getStyles = (props: IDetailsRowStyleProps): IDetailsRowStyles => {
       }
     ],
 
-    cell: defaultCellStyles,
-
+    cell: [defaultCellStyles],
```

Comment:
```suggestion
    cell: defaultCellStyles,
```

Shouldn't need to convert this to an array since it is a single value, please remove